client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -858,18 +838,7 @@ public void onFailure(Exception e) {
         SnapshotLifecyclePolicyMetadata policyMeta =
             getResponse.getPolicies().get("policy_id"); // <1>
         long policyVersion = policyMeta.getVersion();
-        long policyModificationDate = policyMeta.getModifiedDate();
```

Comment:
Even though these appear unused, they are documenting how to retrieve things from the policy metadata, so we should leave them. See `get_snapshot_lifecycle_policy.asciidoc` for the callout usages